gse-app/src/main/java/com/powsybl/gse/app/ProjectPane.java

```diff
@@ -161,6 +167,35 @@ public void contentChanged() {
         };
     }
 
+    private void handleEvent(NodeEvent nodeEvent) {
+        ProjectFile projectFile = project.getFileSystem().findProjectFile(nodeEvent.getId(), ProjectFile.class);
+        if (NodeDataUpdated.TYPENAME.equals(nodeEvent.getType()) || DependencyAdded.TYPENAME.equals(nodeEvent.getType())) {
```

Comment:
check if projectFile's project is the same as this ProjectPane's
we don't want to try refreshing a node that for sure doesn't exist here

Reply:
resolved